### ISSUE TITLE
linux flashing seems to be harcoded

### ISSUE THREAD

**Issue by vpeterhun** [CTCaer/hekate#408]
Hi,

I was trying to flash linux, couldn't figured out, looked at the source.
It seems like the linux image path is hard coded to be "switchroot/install/l4t.00"
I tried it with that path, now it works.

**Comment by CTCaer** [comment]
Yes they are and you can find them in `Note 3` inside the partition manager.

**Comment by vpeterhun** [comment]
Okay thanks, I guess I'm not bright enough to understand the note

**Comment by CTCaer** [comment]
The note effectively says that the Flash options use the directory `switchroot/install/` and read specific filenames in there.
For linux it reads l4t.00, l4t.01, etc. Depending on how big is that distro's rootfs.

**Comment by vpeterhun** [comment]
Yes, now I understand. I don't want to take more of your time.
Initially I thought it meant that XX is a wildcard of any kind. I tried "l4t.img", tried the whole img file with "l4t." prefixed in front of it. Then I tried "l4t.XX" as a filename.
Since I had formatted my card to exFAT it didn't occured to me that XX means parts of the file for fat32.
After that I looked at the code.

Now I'm just trying to figure out how to make it "see" the flashed linux, as I can't seem to be able to boot into it.
